NOTICE

```diff
@@ -1,5 +1,5 @@
 Splunk SOAR RL TitaniumScale Enterprise File Visibility
-Copyright (c) ReversingLabs Inc 2016-2019
+Copyright (c) ReversingLabs Inc 2016-2021
```

Comment:
2022

Reply:
Thanks @ricardo-phantom , updated the year